src/Serilog/Context/ImmutableStack.cs

```diff
@@ -15,22 +15,34 @@
 #if LOGCONTEXT
```

Comment:
If we now have dotnet5.1 TFM support, there's no need for `#if LOGCONTEXT` anymore - that's all the target platforms covered (I hope :-)) - they should all support `LogContext` now in one form or another.